src/Microsoft.DotNet.ProjectModel/Resolution/LibraryManager.cs

```diff
@@ -157,7 +157,7 @@ public IList<DiagnosticMessage> GetAllDiagnostics()
 
                         if (item.Library != library && !versionRange.Satisfies(library.Identity.Version))
                         {
-                            var message = $"Dependency conflict. {item.Library.Identity} expected {FormatLibraryRange(item.Dependency)} but got {library.Identity.Version}";
+                            var message = $"Dependency conflict. {item.Library.Identity} expected {FormatLibraryRange(item.Dependency)} but received {library.Identity.Version}";
```

Comment:
idk if it "received" or "found" it ... I was under the impression that it was being forced to accept a package that it wouldn't have otherwise used. Therefore, it feels like it "expected" to get one thing but "received" something else it didn't want.

:smile: yeah ... I just put this in because of the other more serious issue/question regarding knowing why the downgrade occurred brought it to attention and I thought I would forget. When @blackdwarf put it on the backlog, I thought that was an GREAT idea. This nit means *nothing* really right now.